Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -721,8 +717,8 @@ namespace OpenInfraPlatform {
 
 						if (loopVertices3D.size() < 3) 
 						{
-							BLUE_LOG(warning) << "FaceConverter Problem with Face #" << faceID << ": IfcLoop #" << loop->getId() << " Number of vertices < 2.";
-
+							//BLUE_LOG(warning) << "FaceConverter Problem with Face #" << faceID << ": IfcLoop #" << loop->getId() << " Number of vertices < 2.";
+							throw oip::InconsistentGeometryException(face, " Number of vertices < 2.");

```

Comment:
Technically, this should be tested within the `IfcLoop` converter. But ok, if left like this.

Please provide the `loop` as parameter to the exception, not the face, since the loop is the one having problems.

Reply:
Resolved: a46e819